src/Okta.Idx.Sdk/IIdxContext.cs

```diff
@@ -1,16 +1,18 @@
-﻿// <copyright file="IIdxClientContext.cs" company="Okta, Inc">
+﻿// <copyright file="IIdxContext.cs" company="Okta, Inc">
 // Copyright (c) 2020 - present Okta, Inc. All rights reserved.
 // Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
 // </copyright>
 
 namespace Okta.Idx.Sdk
 {
-    public interface IIdxClientContext
+    public interface IIdxContext
```

Comment:
I see that the context includes the challenge and challenge method. Is this necessary? I don't see it as a required parameter in the `exchangeCode` API call.

Reply:
It's not required, but they are all related to the PKCE details that belong to the Idx Context. Do you foresee any issues by exposing these props?

Reply:
I don't think so, other than it's additional data that would be stored away to be used later.